### DISCUSSION TITLE
Funcionalidad de Resumen diario

### DISCUSSION THREAD

**Discussion by sigfrido45** [thegreenter/greenter#155] [Q&A] [answered]
Saludos, tengo una duda con respecto a la funcionalidad de resumen diario. ¿En qué casos se utilizaría el resumen diario?, no solo se realiza el envío a la sunat mediante los ejemplos mostrados? Osea los ejemplos por defecto que se muestran en la documentación.

**Comment by erajuan** [comment, marked as answer]
Hola @sigfrido45  
El resumen diario se utiliza para enviar las boletas y sus correspondientes notas.

1. En un resumen puedes enviar hasta 500 comprobantes 
2.  Sunat te devuelve un ticket,
3. Validas el estado del ticket
Si envías por resumen las boletas no es necesario enviar de forma individual. Y para enviar una nota primero tienes que enviar la boleta.

**Reply by sigfrido45** [reply]
Gracias por la respuesta, entonces solo aplica para cuando es boleta. Entonces después de generar el xml firmado de una sola boleta puedo generar el resumen diario y enviarlo en una misma petición (por así decirlo) ?

**Reply by erajuan** [reply]
- Aplica para boletas y notas de boletas
- Si, tienes que generar el XML y FIRMAR y almacenarlo
- Si, podrías generar un resumen para solo una boleta. Pero te recomiendo Enviar en grupos de hasta 500 boletas.

**Reply by sigfrido45** [reply]
Gracias, ¿es necesario enviar el ticket una vez enviado el summary para comunicar la boleta emitida? O solo es necesario realizar el summary para comunicar las boletas?. 
Por otro lado, existe una forma de ver los comprobantes de prueba en la SUNAT, trato de visualizarlos mediante la consulta de CPE pero no figura.

**Reply by erajuan** [reply]
1. Al momento de enviar el resumen diario a sunat, sunat te retorna ticket, tienes que validar que el resumen haya sido aceptado/observado enviando el ticket, 
2. No hay forma de verificar comprobantes enviados de forma individual. Pero el ticket de un resumen si puedes validarlo (en prueba).

**Reply by sigfrido45** [reply]
Gracias por las respuestas
